OrangeHRMTests/PageObjects/LoginPage.cs

```diff
@@ -0,0 +1,45 @@
+﻿using OpenQA.Selenium;
+using OrangeHRMTests.Common.Drivers;
+using OrangeHRMTests.Common.WebElements;
+using OrangeHRMTests.Data;
+
+namespace OrangeHRMTests.PageObjects
+{
+    public class LoginPage : BasePage
```

Comment:
I think this page should be in Pages folder

Reply:
Fixed